src/styles/components/header.scss

```diff
@@ -1,2 +1,95 @@
+@import '../mixins';
+
 .header {
+    background: var(--color-background-darker);
+    font-size: 1.6rem;
+    border-bottom: 0.1rem solid #5d5c61;
+    font-weight: var(--weight-semi-bold);
+    
+
+    &__content{
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+    }
+
+    &__content-item{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-shrink: 5;
+        margin-left: 3rem;
+        padding: 15px 0;
+    }
+
+    &__content-item:nth-of-type(2){
+        display: none;
+    }
+
+    &__menu-item{
+        display: inline-block;
+        margin-left: 0.2rem;
+        cursor: pointer;
+        text-decoration: none;
+        transform: translateY(1.3px);
+        
+
+        &:visited{
+            color: var(--color-text-primary);
+        }
+
+        &:first-child{
+            margin-left: 0;
+        }
+
+        &:hover{
+            color: var(--color-text-secondary);
+        }
+    }
+
+    &__logo-link{
+        cursor: pointer;  
+        display: flex;
+        justify-content: center;
+        padding: 5px 0;
+    }
+
+    &__logo{
+        height: 20px;
+        pointer-events: none;
+    }
+
+    &__menu-icon{
+        display: flex;
+        justify-content: center;
+        margin-right: 3rem;
+        animation: menuItemappear 0.2s linear;
+    }
+
+    @include responsive('tablet'){
+        &__menu-item{
+        font-size: 2rem;
+        margin-left: 3rem;
+        animation: menuItemAppear 0.3s linear;
+        }
+
+        &__contentitem:first-child{
+            margin-left: 3rem;
+        }
+
+        &__content-item:nth-of-type(2){
```

Comment:
remove any selector of this type, either use last child or a class for the specific item you are trying to modify

Reply:
Done